meta-python/recipes-devtools/python/python3-geomet_1.1.0.bb

```diff
@@ -12,7 +12,7 @@ S = "${WORKDIR}/git"
 SRC_URI = "git://github.com/geomet/geomet.git;protocol=https;branch=master"
 SRCREV = "6ac73c312b52aca328db2e61d90c5e363b62639f"
 
-inherit setuptools3
+inherit python_flit_core
```

Comment:
Nope, same problem:
```
% rpm -qlp python3-geomet-1.1.0-r0.core2_64.rpm 
/usr
/usr/lib
/usr/lib/python3.10
/usr/lib/python3.10/site-packages
/usr/lib/python3.10/site-packages/geomet-0.0.0.dist-info
/usr/lib/python3.10/site-packages/geomet-0.0.0.dist-info/AUTHORS.txt
/usr/lib/python3.10/site-packages/geomet-0.0.0.dist-info/LICENSE
/usr/lib/python3.10/site-packages/geomet-0.0.0.dist-info/METADATA
/usr/lib/python3.10/site-packages/geomet-0.0.0.dist-info/RECORD
/usr/lib/python3.10/site-packages/geomet-0.0.0.dist-info/WHEEL
/usr/lib/python3.10/site-packages/geomet-0.0.0.dist-info/top_level.txt
```
Even though it shouldn't play a role, note that I'm building under Kirkstone (4.0.19)

Reply:
Using `python_flit_core`, as expected:
```
% rpm -qlp python3-geomet-1.1.0-r0.core2_64.rpm
/usr
/usr/bin
/usr/bin/geomet
/usr/lib
/usr/lib/python3.10
/usr/lib/python3.10/site-packages
/usr/lib/python3.10/site-packages/geomet
/usr/lib/python3.10/site-packages/geomet-1.1.0.dist-info
/usr/lib/python3.10/site-packages/geomet-1.1.0.dist-info/LICENSE
/usr/lib/python3.10/site-packages/geomet-1.1.0.dist-info/METADATA
/usr/lib/python3.10/site-packages/geomet-1.1.0.dist-info/RECORD
/usr/lib/python3.10/site-packages/geomet-1.1.0.dist-info/WHEEL
/usr/lib/python3.10/site-packages/geomet-1.1.0.dist-info/entry_points.txt
/usr/lib/python3.10/site-packages/geomet/__init__.py
/usr/lib/python3.10/site-packages/geomet/__pycache__
/usr/lib/python3.10/site-packages/geomet/__pycache__/__init__.cpython-310.pyc
/usr/lib/python3.10/site-packages/geomet/__pycache__/esri.cpython-310.pyc
/usr/lib/python3.10/site-packages/geomet/__pycache__/geopackage.cpython-310.pyc
/usr/lib/python3.10/site-packages/geomet/__pycache__/tool.cpython-310.pyc
/usr/lib/python3.10/site-packages/geomet/__pycache__/util.cpython-310.pyc
/usr/lib/python3.10/site-packages/geomet/__pycache__/wkb.cpython-310.pyc
/usr/lib/python3.10/site-packages/geomet/__pycache__/wkt.cpython-310.pyc
/usr/lib/python3.10/site-packages/geomet/esri.py
/usr/lib/python3.10/site-packages/geomet/geopackage.py
/usr/lib/python3.10/site-packages/geomet/tests
/usr/lib/python3.10/site-packages/geomet/tests/__init__.py
/usr/lib/python3.10/site-packages/geomet/tests/__pycache__
/usr/lib/python3.10/site-packages/geomet/tests/__pycache__/__init__.cpython-310.pyc
/usr/lib/python3.10/site-packages/geomet/tests/__pycache__/esri_test.cpython-310.pyc
/usr/lib/python3.10/site-packages/geomet/tests/__pycache__/test_cli.cpython-310.pyc
/usr/lib/python3.10/site-packages/geomet/tests/__pycache__/test_geopackage.cpython-310.pyc
/usr/lib/python3.10/site-packages/geomet/tests/__pycache__/util_test.cpython-310.pyc
/usr/lib/python3.10/site-packages/geomet/tests/__pycache__/wkb_test.cpython-310.pyc
/usr/lib/python3.10/site-packages/geomet/tests/__pycache__/wkt_test.cpython-310.pyc
/usr/lib/python3.10/site-packages/geomet/tests/esri_test.py
/usr/lib/python3.10/site-packages/geomet/tests/test_cli.py
/usr/lib/python3.10/site-packages/geomet/tests/test_geopackage.py
/usr/lib/python3.10/site-packages/geomet/tests/util_test.py
/usr/lib/python3.10/site-packages/geomet/tests/wkb_test.py
/usr/lib/python3.10/site-packages/geomet/tests/wkt_test.py
/usr/lib/python3.10/site-packages/geomet/tool.py
/usr/lib/python3.10/site-packages/geomet/util.py
/usr/lib/python3.10/site-packages/geomet/wkb.py
/usr/lib/python3.10/site-packages/geomet/wkt.py
```

Reply:
with `inherit  python_setuptools_build_meta
` I see following in master branch

```
usr/
usr/bin/
usr/bin/geomet
usr/lib/
usr/lib/python3.12/
usr/lib/python3.12/site-packages/
usr/lib/python3.12/site-packages/geomet/
usr/lib/python3.12/site-packages/geomet/__init__.py
usr/lib/python3.12/site-packages/geomet/__pycache__/
usr/lib/python3.12/site-packages/geomet/__pycache__/__init__.cpython-312.pyc
usr/lib/python3.12/site-packages/geomet/__pycache__/esri.cpython-312.pyc
usr/lib/python3.12/site-packages/geomet/__pycache__/geopackage.cpython-312.pyc
usr/lib/python3.12/site-packages/geomet/__pycache__/tool.cpython-312.pyc
usr/lib/python3.12/site-packages/geomet/__pycache__/util.cpython-312.pyc
usr/lib/python3.12/site-packages/geomet/__pycache__/wkb.cpython-312.pyc
usr/lib/python3.12/site-packages/geomet/__pycache__/wkt.cpython-312.pyc
usr/lib/python3.12/site-packages/geomet/esri.py
usr/lib/python3.12/site-packages/geomet/geopackage.py
usr/lib/python3.12/site-packages/geomet/tool.py
usr/lib/python3.12/site-packages/geomet/util.py
usr/lib/python3.12/site-packages/geomet/wkb.py
usr/lib/python3.12/site-packages/geomet/wkt.py
usr/lib/python3.12/site-packages/geomet-1.1.0.dist-info/
usr/lib/python3.12/site-packages/geomet-1.1.0.dist-info/AUTHORS.txt
usr/lib/python3.12/site-packages/geomet-1.1.0.dist-info/LICENSE
usr/lib/python3.12/site-packages/geomet-1.1.0.dist-info/METADATA
usr/lib/python3.12/site-packages/geomet-1.1.0.dist-info/RECORD
usr/lib/python3.12/site-packages/geomet-1.1.0.dist-info/WHEEL
usr/lib/python3.12/site-packages/geomet-1.1.0.dist-info/entry_points.txt
usr/lib/python3.12/site-packages/geomet-1.1.0.dist-info/top_level.txt
```

the backend used by package is also `setuptools.build_meta` see
https://github.com/geomet/geomet/blob/master/pyproject.toml#L5

I am not sure how you are able to build with `flit_core` backend, since it shows python 3.10 paths in
your build, it seems you are not testing the fix on master branch

Reply:
I am building under Yocto 4.0.19 (Kirkstone) by adding `python3-geomet_1.1.0.bb` from master into my own layers. The geomet branch I am using is "release-1.1.0" geomet/geomet@8cdc2e3 that is only one (inconsequential) commit behind master:

```
SRC_URI = "git://github.com/geomet/geomet.git;protocol=https;branch=release-${PV}"
SRCREV = "8cdc2e3c34beb6cbe92efb3086e19950708aed83"
 
inherit python_flit_core
```

I just tried your suggestion using master geomet/geomet@6ac73c3 and switching back to setuptools3:

```
SRC_URI = "git://github.com/geomet/geomet.git;protocol=https;branch=master"
SRCREV = "6ac73c312b52aca328db2e61d90c5e363b62639f"

inherit setuptools3
```

but I see exactly the same problem:

```
% rpm -qlp python3-geomet-1.1.0-r0.core2_64.rpm 
/usr
/usr/lib
/usr/lib/python3.10
/usr/lib/python3.10/site-packages
/usr/lib/python3.10/site-packages/geomet-0.0.0.dist-info
/usr/lib/python3.10/site-packages/geomet-0.0.0.dist-info/AUTHORS.txt
/usr/lib/python3.10/site-packages/geomet-0.0.0.dist-info/LICENSE
/usr/lib/python3.10/site-packages/geomet-0.0.0.dist-info/METADATA
/usr/lib/python3.10/site-packages/geomet-0.0.0.dist-info/RECORD
/usr/lib/python3.10/site-packages/geomet-0.0.0.dist-info/WHEEL
/usr/lib/python3.10/site-packages/geomet-0.0.0.dist-info/top_level.txt
/usr/share
/usr/share/python3-geomet
/usr/share/python3-geomet/LICENSE
/usr/share/python3-geomet/docs
/usr/share/python3-geomet/docs/dev_guide.md
/usr/share/python3-geomet/docs/wkb_detail.md
```